### PR TITLE
Expansion

### PR DESCRIPTION
**Pull request by fluffy-hamster**
Mag, Van, Son, Lyo, Aby, Vet faction branches + fix lock mana

### CODE REVIEW

File: Client/src/main/resources/fx/fx_f5_amplification.plist.json [hunk 1]
```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_f5_amplification.plist.json:1]: change

File: Client/src/main/resources/fx/fx_buff.plist.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "frameWidth": 46,
   "frameHeight": 56,
   "frameDuration": 100,
-  "extraX": 23.0,
-  "extraY": 35.0,
+  "extraX": 0.0,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_buff.plist.json:5]: probably want old values (23, 35)

File: Client/src/main/resources/fx/fx_explosionorangesmoke.plist.json [hunk 3]
```diff
@@ -1,35 +1,103 @@
...
-      },
-      {
-        "x": 0,
-        "y": 0
-      }
-    ]
-  }
-}
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_explosionorangesmoke.plist.json:1]: wrong

File: Client/src/main/resources/fx/fx_f2_eightgates_teallotus.plist.json [hunk 4]
```diff
@@ -1,151 +1,509 @@
...
-      },
-      {
-        "x": 0,
-        "y": 0
-      }
-    ]
-  }
-}
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_f2_eightgates_teallotus.plist.json:1]: wrong

File: Client/src/main/resources/fx/fx_f5_manaburn.plist.json [hunk 5]
```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_f5_manaburn.plist.json:1]: wrong

File: Client/src/main/resources/fx/fx_f5_tremor.plist.json [hunk 6]
```diff
@@ -0,0 +1,467 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_f5_tremor.plist.json:1]: wrong

File: Client/src/main/resources/fx/fx_heavenlystrike.plist.json [hunk 7]
```diff
@@ -2,8 +2,8 @@
   "frameWidth": 80,
   "frameHeight": 240,
   "frameDuration": 100,
-  "extraX": 48.0,
-  "extraY": 275.0,
+  "extraX": 0.0,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_heavenlystrike.plist.json:5]: use old

File: Client/src/main/resources/fx/fx_impact2.plist.json [hunk 8]
```diff
@@ -1,9 +1,9 @@
 {
-  "frameWidth": 50,
-  "frameHeight": 50,
+  "frameWidth": 24,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/fx/fx_impact2.plist.json:2]: use old ??

File: Client/src/main/resources/icons/icon_f5_bloodrage.plist.json [hunk 9]
```diff
@@ -0,0 +1,341 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/icons/icon_f5_bloodrage.plist.json:1]: wrong

File: Client/src/main/resources/icons/icon_f5_dampeningwave.plist.json [hunk 10]
```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/icons/icon_f5_dampeningwave.plist.json:1]: wrong

File: Client/src/main/resources/icons/icon_f5_diretidefrenzy.plist.json [hunk 11]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/icons/icon_f5_diretidefrenzy.plist.json:1]: wrong

File: Client/src/main/resources/icons/icon_f5_greaterfortitude.plist.json [hunk 12]
```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/icons/icon_f5_greaterfortitude.plist.json:1]: wrong

File: Client/src/main/resources/icons/icon_f5_spiritofvaath.plist.json [hunk 13]
```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fluffy-hamster) [inline Client/src/main/resources/icons/icon_f5_spiritofvaath.plist.json:1]: wrong

File: Client/src/main/resources/troopAnimations/f3_buildcommon.plist.json [hunk 14]
```diff
@@ -2,8 +2,8 @@
   "frameWidth": 120,
   "frameHeight": 120,
   "frameDuration": 100,
-  "extraX": 75.0,
-  "extraY": 75.0,
+  "extraX": 100.0,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/troopAnimations/f3_buildcommon.plist.json:5]: which one is good??

File: Client/src/main/resources/troopAnimations/f3_mech.plist.json [hunk 15]
```diff
@@ -2,8 +2,8 @@
   "frameWidth": 110,
   "frameHeight": 110,
   "frameDuration": 100,
-  "extraX": 75.0,
-  "extraY": 75.0,
+  "extraX": 85.0,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/troopAnimations/f3_mech.plist.json:5]: use new ??

File: Client/src/main/resources/troopAnimations/f3_windstriker.plist.json [hunk 16]
```diff
@@ -3,7 +3,7 @@
   "frameHeight": 100,
   "frameDuration": 100,
   "extraX": 75.0,
```
Comment (fluffy-hamster) [inline Client/src/main/resources/troopAnimations/f3_windstriker.plist.json:5]: use new ??

File: devTools/cardCreator/cards_to_import/ShadowHawk.minion.card.json [hunk 17]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (fluffy-hamster) [inline devTools/cardCreator/cards_to_import/ShadowHawk.minion.card.json:1]: maybe a fuckup

File: devTools/cardCreator/cards_to_import/NeedValidationChecks/Heartseeker.minion.card.json [hunk 18]
```diff
@@ -1,17 +0,0 @@
-{
```
Comment (fluffy-hamster) [inline devTools/cardCreator/cards_to_import/NeedValidationChecks/Heartseeker.minion.card.json:1]: mistakenly deleted

File: Server/resources/spellCards/Gotatsu.spell.card.json [hunk 19]
```diff
@@ -1,16 +1,19 @@
 {
-  "name": "Fireball",
-  "description": "Deal 3 damage to anything.",
-  "cardId": "Fireball",
-  "spriteName": "icon_f2_fireball",
+  "name": "Gotatsu",
```
Comment (fluffy-hamster) [inline Server/resources/spellCards/Gotatsu.spell.card.json:2]: another fuckup, probably from merging songhai

**Comment by Ridleh** [comment]
on my pc collection screen takes ~8 seconds to open and fps drops to ~15. ram usage jumps up to 1 gig and above. 

would not recommend we merge until we get more people to test this branch out and optimize/refactor collection screen

**Pull request closed without merging**